_pages/developers.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Developers
+permalink: /developers/
+---
+
+<div class="bg-navy">
+  <div class="container cntnr-wide px2 py5 clearfix">
+    <img alt="" width="231" class="mt1 mx4 right md-show" src="{{ '/assets/img/playbook-landing.svg' | prepend: site.baseurl }}">
+    <h1 class="mt0 mb1 white">
+      Built for easy integration
+    </h1><img alt="" class="mb3" src="{{ '/assets/img/hr-red-2.svg' | relative_url }}">
+    <p class="mb3 overflow-hidden white fs-20p serif">
+      We speed up integration with open code repositories, standards-based support, and API-based interactions.
+    </p>
+    <a href="https://developers.login.gov/" target="_blank" class="btn btn-primary btn-wide">Get started</a>
+  </div>
+</div>
+
+<div class="bg-white">
+  <div class="container cntnr-wide p2">
+
+    <div class="center mt3 mb4">
+      <h2 class="h1 mt0 mb1">Built for flexibility</h2>
+      <p class="mb0 fs-20p">
+       We offer options for different apps
+
+      </p>
+    </div>
+
+    <div class="clearfix mxn2">
+      <div class="col col-12 sm-col-4 px2 mb3 center">
+        <img alt="" src="{{ site.baseurl }}/assets/img/users.svg" height="90">
+        <h2 class="mt2 mb2 blue"><a href="https://developers.login.gov/" target="_blank" class="text-decoration-none">Comply with changing identity assurance guidelines</a></h2>
+        <p>We currently support LOA1 and LOA3 for NIST 800-63-2, and are updating for compliance with 800-63-3.</p>
+      </div>
+      <div class="col col-12 sm-col-4 px2 mb3 center">
+        <img alt="" src="{{ site.baseurl }}/assets/img/partners.svg" height="90">
+        <h2 class="mt2 mb2 blue"><a href="https://developers.login.gov/" target="_blank" class="text-decoration-none">Get the flexibility to adapt</a></h2>
+        <p>Major authentication protocols such as OpenID and SAML have reusable libraries behind them, which reduce the amount of coding necessary for you to adopt them.</p>
```

Comment:
`Major authentication protocols such as OpenID and SAML have reusable libraries behind them, which reduce the amount of coding necessary for you to adopt them.`

And then shorten this to...

`We support standard protocols such as OpenID and SAML, letting you take full advantage of reusable code libraries.`